docs/farming-&-staking/staking/operators/tips-operator.mdx

```diff
@@ -13,34 +13,20 @@ import CodeBlock from '@theme/CodeBlock';
 import Link from '@docusaurus/Link';
 import styles from '@site/src/pages/index.module.css';
 
-### Create operator key (alternative, less secure way):
-An operator needs a key pair to participate in bundle production.
-You can create a key using the following command:
-
-```bash
-target/production/subspace-node key generate --scheme sr25519
-```
-
-![Staking-4](/img/doc-imgs/operators-staking/Staking-4.png)
-
-Back up the key. Take the `public key (hex)` of the Keypair. The public key is part of the operator config we will be using later on [Staking portal](https://staking.subspace.tools) or [PolkadotJS portal](https://polkadot.js.org/apps/#/explorer).
 
 #### Insert key to Keystore:
-The key generated above needs to be added to the Keystore so that the operator node can use it to participate in bundle production.
+This might be useful if you decicded to switch domains and already have **the secret phrase**. Read more about switching domain in the next section.
 
 To insert the key, use the following command:
 
 ```bash
-target/production/subspace-node key insert \
---suri "<Secret phrase>" --key-type oper --scheme sr25519 --keystore-path /keystore
+target/production/subspace-node domain key insert \
+--base-path PATH_TO_NODE --domain-id DOMAIN_ID --keystore-suri "<Secret phrase>" --key-type oper --scheme sr25519
```

Comment:
This is incorrect, there is no `--key-type` or `--scheme` options. We only support very specific keys and our custom commands uses them implicitly without need/ability to specify.